src/lib/select/option.ts

```diff
@@ -1,10 +1,82 @@
-import {Component, ViewEncapsulation} from '@angular/core';
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  Output,
+  Renderer,
+  ViewEncapsulation
+} from '@angular/core';
+import {ENTER, SPACE} from '../core/keyboard/keycodes';
 
 @Component({
   moduleId: module.id,
   selector: 'md-option',
-  template: ``,
+  host: {
+    'role': 'option',
+    '[class.md-selected]': 'selected',
+    '[attr.aria-selected]': 'selected.toString()',
+    '[attr.tabindex]': '0',
```

Comment:
Yes, but I'll need to bind it again as soon as I add disabled support (in the next PR probably)